src/viam/services/slam/slam.py

```diff
@@ -47,6 +47,12 @@ async def get_point_cloud_map(self, return_edited_map: Optional[bool] = None, *,
             # Get the point cloud map in standard PCD format.
             pcd_map = await slam_svc.get_point_cloud_map()
 
+        Args:
+            timeout (Optional[float]): An option to set how long to wait (in seconds)
+                before calling a time-out and closing the underlying RPC call.
```

Comment:
We can get rid of the `timeout` docstring, just the `return_edited_map` is fine :)